### ISSUE TITLE
[BUG] Voteparty Issues

### ISSUE THREAD

**Issue by pjindras** [GeorgeV220/VoteRewards#127]
Hello,
Sometimes it happens, that some player, who voted previously and is online now, doesn't receive the reward.

My config seems like this:

https://pastebin.com/MR5BYiaK
I am using Mysql, server is in offline mode.

Players really voted, but didnt get the reward:

[18:59:08] [Votifier epoll worker/INFO]: [Votifier] Got a protocol v2 vote record from /82.208.16.75:34988 -> Vote (from:Craftlist.org username:Todonka address:86.49.63.96 timeStamp:1684256518470 additionalData:null)
[18:59:08] [Server thread/INFO]: [VoteRewards] [Debug] [Version: 8.3.0] {Craftlist.org=1684256348179, Czech-Craft.eu=1684256335429}
[18:59:08] [Server thread/INFO]: [VoteRewards] [Debug] [Version: 8.3.0] RUNNING COMMANDS FOR PLAYER: Todonka

Please help, thank you.

**Server Information (please complete the following information):**
- Server version: This server is running Paper version git-Paper-448 (MC: 1.19.3) (Implementing API version 1.19.3-R0.1-SNAPSHOT) (Git: 155aa36)
You are running the latest version

- Installed plugins:
- 19:18:59[INFO] Bukkit Plugins:
19:18:59[INFO] - ActionHealth, AdvancedMobArena, ArmorStandTools, *asCMD, AuctionHouse, AuthMe, *BCommons, BeastTokens, BetterRTP, ChatFilter
19:18:59[INFO] ChestShop, Citizens, CMILib, CoreProtect, CraftEnhance, DecentHolograms, DeluxeMenus, DiscordSRV, DropEdit2, DropHeads
19:18:59[INFO] dynmap, Essentials, EssentialsChat, EssentialsProtect, EssentialsSpawn, ExcellentCrates, FastAsyncWorldEdit, FastLogin, FireMysteryBlocks, GSit
19:18:59[INFO] HeadDatabase, *HostifyMonitor, InventoryRollbackPlus, Jobs, LibsDisguises, LiteBans, LuckPerms, MessageAnnouncer, MineableSpawners, MineMarket
19:18:59[INFO] MMOItems, Multiverse-Core, MyCommand, MythicLib, MythicMobs, NexEngine, *PixelMOTD, PL-Hide, PlaceholderAPI, Plan
19:18:59[INFO] PlugManX, ProtocolLib, Quests, QuestsBar, Residence, ShopGUIPlus, ShopGUIPlusMineableSpawnersBridge, SkinsRestorer, sleep-most, TAB
19:18:59[INFO] TitleManager, TradeSystem, UltimateRewards, UltimateTimber, Vault, ViaBackwards, ViaVersion, VoidGen, VoteRewards, Votifier

**Comment by GeorgeV220** [comment]
From what I understand, sometimes a player who did an offline vote when connected to the server does not receive the rewards?

**Comment by pjindras** [comment]
> From what I understand, sometimes a player who did an offline vote when connected to the server does not receive the rewards?

No, sometimes player, who voted doesnt get voteparty reward, but the player is online when the voteparty is made

**Comment by pjindras** [comment]
It also happened, that the player, who voted received the reward, but also received the message, that he didn't voted so hi will not receive the reward, if you know what i mean, the reward was 2k in cash, see the screenshot. (it is different player than who voted on screenshot)

![image](https://github.com/GeorgeV220/VoteRewards/assets/45400186/1917770f-ca4e-43bf-b34a-9d4cac87a22e)

**Comment by GeorgeV220** [comment]
I'll look into it
